mmselfsup/datasets/transforms/processing.py

```diff
@@ -1255,3 +1255,123 @@ def __repr__(self) -> str:
         if self.fill is not None:
             repr_str += f', fill={self.fill})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class MaskFeatMaskGenerator(BaseTransform):
```

Comment:
Maybe should reuse [BEiTMaskGenerator](https://github.com/open-mmlab/mmselfsup/blob/cbe458bb07d688a802557cf0c5c712e82b0992c2/mmselfsup/datasets/transforms/processing.py#L107)